scripts/who/gho/generate_csv_and_sv.py

```diff
@@ -0,0 +1,308 @@
+            indicators: map of indicator codes to its corresponding schema dcid
+      }
+      output_dir: directory to output the csv and stat var mcf to
+  """
+    seen_sv = set()
+    mcf_result = []
+    cprop_mapping = schema_mapping.get("cprops", {})
+    value_mapping = schema_mapping.get("values", {})
+    indicator_mapping = schema_mapping.get("indicators", {})
+    person_dimensions = schema_mapping.get("personDimensions", {})
+    with open(os.path.join(output_dir, "who.csv"), "w+") as csv_file:
+        writer = csv.writer(csv_file)
+        writer.writerow(["year", "period", "country", "statVar", "value"])
+        for f in data_files:
+            with open(f, "r+") as indicator_values:
+                indicator_values = json.load(indicator_values).get("value", [])
+            value_map = {}
+            for entry in indicator_values:
+                time_dim_value = entry.get("TimeDim")
+                if not time_dim_value:
+                    continue
+                time_period = get_time_period(entry)
+                spatial_dim_type = entry.get("SpatialDimType")
+                if not spatial_dim_type == "COUNTRY":
+                    continue
+                spatial_dim = entry.get("SpatialDim")
+                i_code = entry.get("IndicatorCode", "")
+                sv_dcid = i_code
+                measured_prop = indicator_mapping.get(i_code, i_code)
+                mcf = [
+                    "typeOf: dcs:StatisticalVariable",
+                    f"measuredProperty: dcs:who/{i_code}"
+                ]
+                value, statType = get_value(entry)
+                mcf.append(f"statType: dcs:{statType}")
+                if not value or value in VALUES_TO_DROP:
+                    continue
+                sv_dcid, has_empty_mapped_val = process_dimensions(
+                    cprop_mapping, value_mapping, person_dimensions, entry, mcf,
+                    sv_dcid)
+                if not sv_dcid in seen_sv:
+                    value_map[sv_dcid] = {}
+                    mcf.insert(0, f"Node: dcid:{sv_dcid}")
+                    mcf_result.extend(mcf + [""])
+                seen_sv.add(sv_dcid)
+                map_key = f"{spatial_dim}^{time_dim_value}^{time_period}"
+                if not map_key in value_map[sv_dcid]:
+                    value_map[sv_dcid][map_key] = []
+                comments_length = 0
+                if entry.get("Comments", ""):
+                    comments_length = len(entry.get("Comments"))
+                new_value_entry = {
+                    "spatial_dim": spatial_dim,
+                    "time_dim_value": time_dim_value,
+                    "time_period": time_period,
+                    "value": value,
+                    "date": entry.get("Date", ""),
+                    "commentsLength": comments_length,
+                    "hasEmptyDimVal": has_empty_mapped_val
+                }
```

Comment:
comment about the need for commentsLength, date here?  (As I recall, to avoid duplicate obs vaues?)

Reply:
done